src/Coord.py

```diff
@@ -0,0 +1,18 @@
+import chess_enums
+
+
+class Action:
+    c: int
+    r: int
+
+    def __init__(self, par_space: chess_enums.Space) -> None:
+        self.c = par_space / 8
+        self.r = par_space % 8
+
+    def __init__(self, par_c: int, par_r: int) -> None:
```

Comment:
I don't think you can define multiple __init__ functions. When I looked it up, I read that it won't give you an error, but only one of the definitions will be accepted. There are a couple ways around this. I think the way that seemed most appealing to me was to have the __init__ function take row and column, and then have a classmethod called FromSpace that takes a Space, and that function would just return the r,c version. When I tried it it look like "def FromSpace(cls, par_space): return cls(<col>,<row>)

Reply:
Ah I see what you mean, I have removed the second __init__ and created fromSpace